code/modules/mob/living/living.dm

```diff
@@ -892,6 +892,10 @@
 		var/obj/mecha/M = loc
 		loc_temp =  M.return_temperature()
 
+	else if(istype(loc, /obj/spacepod))
```

Comment:
oh god
i-i think i am going to puke

whhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy